notebooks/official/training/multi_node_ddp_nccl_vertex_training_with_custom_container.ipynb

```diff
@@ -1076,7 +950,11 @@
         "id": "535932b8c42d"
       },
       "source": [
-        "## Executes in Workbench\n"
+        "## Executes in a notebook environment\n",
+        "\n",
+        "If you're running this tutorial in a notebook environment like Vertex AI workbench, proceed with the below steps. \n",
```

Comment:
change "like" to "for example":

- If you're running this tutorial in a notebook environment, for example Vertex AI workbench, proceed with the below steps.